Localisation/Editor/LocalisationEditor.cs

```diff
@@ -79,8 +79,9 @@ private void OnGUI()
 			}
 			else
 			{
+				var numberOfCategories = (localSchema.categories == null) ? 0 : localSchema.categories.Length;
```

Comment:
Fixed a null-ref exception as I needed it to continue working (although ideally this should be in the loc branch - see other PR)